pgbouncer/datadog_checks/pgbouncer/pgbouncer.py

```diff
@@ -164,11 +164,17 @@ def _get_connection(self, use_cached=None):
         use_cached = use_cached if use_cached is not None else self.use_cached
```

Comment:
```suggestion
        if use_cached is None:
        	use_cached = self.use_cached
```
Can we simplify this for clarity?

Reply:
I refactored away this part - see the comment in the main thread for the PR!